### PR TITLE
feat: noCommit input

### PR DESCRIPTION
**Pull request by dsanders11**
Feature to not commit and push, only stage the changes. This leaves the checkout in the ready to commit state which allows another action ([`dsanders11/github-app-commit-action`](https://github.com/dsanders11/github-app-commit-action)) to do the commit and get verified commits via GitHub's API.

### CODE REVIEW

**Comment by dsanders11** [comment]
@malept, here's the draft PR for how this will be used: https://github.com/electron/get/pull/276

File: entrypoint.sh [hunk 1]
```diff
@@ -86,22 +86,28 @@ fi
...
-    git commit $INPUT_GITCOMMITFLAGS --message "$INPUT_GITCOMMITMESSAGE"
-
-    if test -n "$GH_PAGES_SSH_DEPLOY_KEY"; then
-        mkdir ~/.ssh
-        echo "$GH_PAGES_SSH_DEPLOY_KEY" > ~/.ssh/deploy_key
-        chmod 400 ~/.ssh/deploy_key
-
-        git remote rm origin
-        git remote add origin "git@github.com:$GITHUB_REPOSITORY.git"
+if test -z "$INPUT_NOCOMMIT"; then
```
Comment (malept) [inline entrypoint.sh]: suggestion: this effectively says "if not no-commit", I think we should flip the conditional for better readability.

**Review by malept — CHANGES_REQUESTED** [review]
This seems fine from a high level but I had one suggestion about readability.

File: entrypoint.sh (hunk 1 above)
Comment (dsanders11) [reply entrypoint.sh]: Good point! Change made.

**Review by malept — APPROVED** [review]

**Pull request merged**